openfl-tutorials/interactive_api/Tensorflow_Word_Prediction/envoy_folder/requirements.txt

```diff
@@ -0,0 +1,3 @@
+gdown==3.13.0
+pyarrow==5.0.0
+tensorflow==2.5.1
```

Comment:
It's experiment-dependent requirement. Should it be in envoy's requirements?

Reply:
It is required by shard descriptor, not by whole experiment.

Reply:
Can we avoid it? Shard descriptor uses only `from tensorflow.keras.utils import to_categorical`

Reply:
Yes, I've changed it to numpy.